golem_messages/datastructures/tasks.py

```diff
@@ -117,10 +117,10 @@ def sign(self, private_key: bytes) -> None:
             msghash=self.get_hash(),
         )
 
-    def verify(self, public_key: bytes) -> None:
+    def verify(self, public_key: bytes) -> bool:
```

Comment:
@shadeofblue this suggests that when verification fails it will return False. :(